test/tasks/ods-build-typescript_test.go

```diff
@@ -34,6 +34,7 @@ func TestTaskODSBuildTypescript(t *testing.T) {
 						filepath.Join(pipelinectxt.CodeCoveragesPath, "lcov.info"),
 						filepath.Join(pipelinectxt.SonarAnalysisPath, "analysis-report.md"),
 						filepath.Join(pipelinectxt.SonarAnalysisPath, "issues-report.csv"),
+						filepath.Join(pipelinectxt.SonarAnalysisPath, "quality-gate.json"),
```

Comment:
`sonar-quality-gate` is not enabled on this test (*build typescript app*), but it is set to true on the following test (*build typescript app in subdirectory*).
Shouldn't the artifact be checked on the test where there is this quality gate rather than the one without it?

Reply:
Thanks. That was a copy-paste error. I have moved the check to the other test in 239a7c9.